### PR TITLE
jbranchaud/er 16 bootstrap skill stack app for epic react

### PR DESCRIPTION
**Pull request by jbranchaud**
_update_: the Mux assets have been uploaded along with SRT tracks, the Sanity import has been tested several times, undergone several refactorings to get the imported data just right, and besides a final sign off on the data that currently exists in Sanity `epic-react/data-import`, this should be good to go.

---

Once we have a Mux environment for Epic React, the Mux asset upload
script can be run. This will produce a JSON output with all the Mux
video identifiers.

The Sanity import script is still somewhat WIP in that it needs to be
run against a non-prod Sanity environment and evaluated for correctness,
but it is probably 95% of the way there. The Mux JSON output is needed
to do that.

- feat(kcd): bootstrap the Epic React app
- feat(kcd): better env var types, get latest mux
- feat(kcd): add mux and sanity import scripts

kirby

### CODE REVIEW

File: apps/epic-react/.env.production [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# ConvertKit Newsletter Form
+NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM=FORM_ID
+NEXT_PUBLIC_CONVERTKIT_TOKEN=CK_TOKEN
+NEXT_PUBLIC_CONVERTKIT_ANSWER_URL=/api/skill/answer/convertkit
+NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL=/api/skill/subscribe/convertkit
+NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL=/api/skill/subscriber/convertkit
+
+# Sanity
+NEXT_PUBLIC_SANITY_API_VERSION=2023-01-03
+NEXT_PUBLIC_SANITY_PROJECT_ID=project_id_from_sanity
```
Comment (jbranchaud) [inline apps/epic-react/.env.production:32]: replace this and `.env.development` with the project ID for the Epic React sanity project once that is setup

File: apps/epic-react/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: '3.8'
+services:
+  db:
+    image: mysql:8.0
+    cap_add:
+      - SYS_NICE
+    restart: always
+    environment:
+      - MYSQL_DATABASE=epic-react
```
Comment (jbranchaud) [inline apps/epic-react/docker-compose.yml:9]: wrong database,

```suggestion
      - MYSQL_DATABASE=kcd-products
```

File: apps/epic-react/schemas/index.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
+// —— documents
+import cta from './documents/cta'
+import exercise from './documents/exercise'
+import explainer from './documents/explainer'
+import interview from './documents/interview'
```
Comment (Creeland) [inline apps/epic-react/schemas/index.ts:5]: It looks like the interview document doesn't actually exist

Comment (jbranchaud) [reply apps/epic-react/schemas/index.ts:5]: Thanks for catching this, updated the PR to include that file.

**Review by Creeland — APPROVED** [review]
💯

**Pull request merged**